### DISCUSSION TITLE
Suboptimal response for regex search

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#14827] [Other Feature Feedback, Questions, & Ideas] [answered]
https://cs.github.com/?scopeName=All+repos&scope=&q=%2F%5Cbsate%5Cb%2F

<img width="923" alt="image" src="https://user-images.githubusercontent.com/2119212/163077793-0b1d0721-68e4-47cc-8a2d-d278e41d61e5.png">

I was at least hoping for something after the `:`...

https://www.githubstatus.com/ is healthy.

Fwiw, the shards don't like me:
https://cs.github.com/?q=sate&p=2&pt=06df1a5300d08874a455343a428d2eb971dda78feb8b0ba812cae749766560630e04c98f7988c1680b67fb02e99675adcb630b181b37927b2d86af59ecfceb2cdbcef96270acf5fd6274fc730a772f987f5e0bba497146ba0033e23b9b001f2012a59a8f71b936e21b6feeb8cf4c52d9d8ddd2835e1001fe21a1ee8c7549d184dfe4e0bead9f968786d179bcc5634893858e71c197d460a1c68e3e2a50b3c83a7094a781f3a97aba988cc9369078192971a045d7f43e9953e6d9ca3b498689890e6f3a6878666ae4b5ddc6f071e820c72f3a7ec525fef74a218dda7e5bb343119144a7aab9bbdb80fe2ded0ca8efcebdbbd231cc7e1026b6e4917dd1a7aa95b25dd8a24eda0b559cc4129fa0c71070d03e24ddb15f4d2a0630fd6f953a5c032c843cb1244bb8812e4b9b3370ff550f3053f6441e59325b9642c395f882bbb8530ed81a103f14e4aeb0979f72cbc57cebe8a910bb3d3bfb883aafd3db24b4589e5248d7634b2d4f2c257e8c7ff07b1237b46e42ebaab59ce34cab70b6a968cf6e753ce12491040faea46a310510e4937a&scope=&scopeName=All+repos

<img width="529" alt="image" src="https://user-images.githubusercontent.com/2119212/163077887-97732716-a060-4e99-b3c8-270fb560cc1d.png">

**Comment by colinwm** [comment, marked as answer]
Interesting find, this is a bug. I'll look into it...

**Reply by Gknhvhhj** [reply]
Giang0909a

**Reply by aneubeck** [reply]
@colinwm Could this be a problem with timeout handling? We definitely found "some" results. But, for whatever reason we don't show those with some indication that we gave up finding more.

**Comment by Gknhvhhj** [comment]
Giang0909a
